source/uwp/Renderer/lib/Util.cpp

```diff
@@ -370,104 +367,110 @@ HRESULT GenerateActionProjection(
 
     switch (action->GetElementType())
     {
-        case ActionType::OpenUrl:
-            RETURN_IF_FAILED(MakeAndInitialize<::AdaptiveNamespace::AdaptiveOpenUrlAction>(projectedAction,
-                std::AdaptivePointerCast<AdaptiveSharedNamespace::OpenUrlAction>(action)));
-            break;
-        case ActionType::ShowCard:
-            RETURN_IF_FAILED(MakeAndInitialize<::AdaptiveNamespace::AdaptiveShowCardAction>(projectedAction,
-                std::AdaptivePointerCast<AdaptiveSharedNamespace::ShowCardAction>(action)));
-            break;
-        case ActionType::Submit:
-            RETURN_IF_FAILED(MakeAndInitialize<::AdaptiveNamespace::AdaptiveSubmitAction>(projectedAction,
-                std::AdaptivePointerCast<AdaptiveSharedNamespace::SubmitAction>(action)));
-            break;
-        case ActionType::Custom:
-            RETURN_IF_FAILED(std::AdaptivePointerCast<CustomActionWrapper> (action)->GetWrappedElement(projectedAction));
-            break;
-        default:
-            return E_UNEXPECTED;
-            break;
+    case ActionType::OpenUrl:
+        RETURN_IF_FAILED(MakeAndInitialize<::AdaptiveNamespace::AdaptiveOpenUrlAction>(projectedAction,
+                                                                                                                                                                     std::AdaptivePointerCast<AdaptiveSharedNamespace::OpenUrlAction>(action)));
+        break;
+    case ActionType::ShowCard:
+        RETURN_IF_FAILED(MakeAndInitialize<::AdaptiveNamespace::AdaptiveShowCardAction>(projectedAction,
+                                                                                                                                                                        std::AdaptivePointerCast<AdaptiveSharedNamespace::ShowCardAction>(action)));
+        break;
+    case ActionType::Submit:
+        RETURN_IF_FAILED(MakeAndInitialize<::AdaptiveNamespace::AdaptiveSubmitAction>(projectedAction,
+                                                                                                                                                                    std::AdaptivePointerCast<AdaptiveSharedNamespace::SubmitAction>(action)));
+        break;
+    case ActionType::Custom:
+        RETURN_IF_FAILED(std::AdaptivePointerCast<CustomActionWrapper>(action)->GetWrappedElement(projectedAction));
+        break;
+    default:
+        return E_UNEXPECTED;
+        break;
     }
 
     return S_OK;
-} CATCH_RETURN;
+}
+CATCH_RETURN;
 
 HRESULT GenerateColumnsProjection(
-    const std::vector<std::shared_ptr<AdaptiveSharedNamespace::Column>>& containedElements,
-    ABI::Windows::Foundation::Collections::IVector<ABI::AdaptiveNamespace::IAdaptiveColumn*>* projectedParentContainer) noexcept try
+        const std::vector<std::shared_ptr<AdaptiveSharedNamespace::Column>> &containedElements,
```

Comment:
>         [](start = 0, length = 8)

Why this change of style to indent the parameters 8 spaces instead of 4? Most of our code indents these 4 spaces.